Swift 5.9 warning about Hashable being deprecated

Generate swift code has compilation warning about deprecated protocol.

```
SamSchema.swift:5645:16: warning: 'Hashable.hashValue' is deprecated as a protocol requirement; conform type 'JSONNull' to 'Hashable' by implementing 'hash(into:)' instead
    public var hashValue: Int {
```

Using : 

```
swift --version 
swift-driver version: 1.87.1 Apple Swift version 5.9 (swiftlang-5.9.0.128.108 clang-1500.0.40.1)
Target: arm64-apple-macosx14.0
```

Steps to reproduce : 

```
curl -o sam.schema.json https://raw.githubusercontent.com/aws/serverless-application-model/develop/schema_source/sam.schema.json

quicktype -s schema sam.schema.json -o SamSchema.swift

swift SamSchema.swift 
SamSchema.swift:5645:16: warning: 'Hashable.hashValue' is deprecated as a protocol requirement; conform type 'JSONNull' to 'Hashable' by implementing 'hash(into:)' instead
    public var hashValue: Int {
```

Expected behaviour : generated code is updated to not include deprecated features.

Thanks !

--- a/packages/quicktype-core/src/language/Swift/SwiftRenderer.ts
+++ b/packages/quicktype-core/src/language/Swift/SwiftRenderer.ts
@@ -1185,12 +1185,10 @@ encoder.dateEncodingStrategy = .formatted(formatter)`);
 			return 0
 	}`);
 
-                if (this._options.swift5Support) {
-                    this.ensureBlankLine();
-                    this.emitMultiline(`    public func hash(into hasher: inout Hasher) {
+                this.ensureBlankLine();
+                this.emitMultiline(`    public func hash(into hasher: inout Hasher) {
 			// No-op
 	}`);
-                }
             }
 
             this.ensureBlankLine();

--- a/test/unit/swift-json-null-hashable.test.ts
+++ b/test/unit/swift-json-null-hashable.test.ts
@@ -1,0 +1,27 @@
+import {
+    InputData,
+    JSONSchemaInput,
+    quicktype,
+} from "../../packages/quicktype-core/src/index.js";
+import { expect, test } from "vitest";
+
+const schema = JSON.stringify({
+    type: "object",
+    properties: {
+        value: {},
+    },
+    required: ["value"],
+});
+
+test("emits hash(into:) for JSONNull by default", async () => {
+    const schemaInput = new JSONSchemaInput(undefined);
+    await schemaInput.addSource({ name: "TopLevel", schema });
+
+    const inputData = new InputData();
+    inputData.addInput(schemaInput);
+
+    const result = await quicktype({ inputData, lang: "swift" });
+    const output = result.lines.join("\n");
+
+    expect(output).toContain("public func hash(into hasher: inout Hasher)");
+});

